### DISCUSSION TITLE
Importing blender files to stride

### DISCUSSION THREAD

**Discussion by Schwert1602** [stride3d/stride#2025] [Newcomers]
Hello,

 I'm new to stride and I am trying to import a rigged 3d model from Blender to stride. When I import it as blender file, I constantly get this error message:

```
`Error: Assimp: Constructing BlenderDNA Structure encountered an error: BlendDNA: Did not find a field named `no` in structure `MVert`
Info: Nothing to import.`
```

I can't export and import it as fbx file because the 3d model consist of multiple objects which I want to stay separate (stride imports them as a single model).

So, is there a way to import .blend files directly? If not, is there another way to import 3d models out of multiple objects with textures from blender to stride?

**Comment by DJDQfff** [comment]
You need to use old version blender.
